--- Python.sublime-syntax.orig  2017-12-01 13:00:22.000000000 +0300
+++ Python.sublime-syntax 2017-12-01 13:02:27.000000000 +0300
@@ -1232,12 +1232,7 @@
             - match: '"""'
               scope: punctuation.definition.string.end.python
               set: after-expression
-            - match: ''
-              push: scope:source.regexp.python
-              with_prototype:
-                - match: '(?=""")'
-                  pop: true
-                - include: escaped-unicode-char
+            - include: escaped-unicode-char
     # Triple-quoted raw string, bytes, will use regex
     - match: '([bB]r|r[bB])(""")'
       captures:
@@ -1248,9 +1243,6 @@
         - match: '"""'
           scope: punctuation.definition.string.end.python
           set: after-expression
-        - match: ''
-          embed: scope:source.regexp.python
-          escape: (?=""")
     # Triple-quoted raw f-string
     - match: ((?i)fr|rf)(""")
       captures:
@@ -1370,12 +1362,6 @@
           scope: punctuation.definition.string.end.python
           set: after-expression
         - include: line-continuation-inside-string
-        - match: ''
-          push: scope:source.regexp.python
-          with_prototype:
-            - match: '(?="|\n)'
-              pop: true
-            - include: line-continuation-inside-string
     # Single-line raw string, bytes, treated as regex
     - match: '([bB]r|r[bB])(")'
       captures:
@@ -1387,9 +1373,6 @@
           scope: punctuation.definition.string.end.python
           set: after-expression
         - include: line-continuation-inside-string
-        - match: ''
-          embed: scope:source.regexp.python
-          escape: (?="|\n)
     # Single-line raw f-string
     - match: ((?i)fr|rf)(")
       captures:
@@ -1512,12 +1495,7 @@
             - match: "'''"
               scope: punctuation.definition.string.end.python
               set: after-expression
-            - match: ''
-              push: scope:source.regexp.python
-              with_prototype:
-                - match: (?=''')
-                  pop: true
-                - include: escaped-unicode-char
+            - include: escaped-unicode-char
     # Triple-quoted raw string, bytes, will use regex
     - match: ([bB]r|r[bB])(''')
       captures:
@@ -1528,9 +1506,6 @@
         - match: "'''"
           scope: punctuation.definition.string.end.python
           set: after-expression
-        - match: ''
-          embed: scope:source.regexp.python
-          escape: (?=''')
     # Triple-quoted raw f-string
     - match: ((?i)fr|rf)(''')
       captures:
@@ -1650,12 +1625,6 @@
           scope: punctuation.definition.string.end.python
           set: after-expression
         - include: line-continuation-inside-string
-        - match: ''
-          push: scope:source.regexp.python
-          with_prototype:
-            - match: '(?=''|\n)'
-              pop: true
-            - include: line-continuation-inside-string
     # Single-line raw string, bytes, treated as regex
     - match: '([bB]r|r[bB])('')'
       captures:
@@ -1667,12 +1636,6 @@
           scope: punctuation.definition.string.end.python
           set: after-expression
         - include: line-continuation-inside-string
-        - match: ''
-          push: scope:source.regexp.python
-          with_prototype:
-            - match: '(?=''|\n)'
-              pop: true
-            - include: line-continuation-inside-string
     # Single-line raw f-string
     - match: ((?i)fr|rf)(')
       captures:
