--- Default (Windows).sublime-keymap
+++ Default (Windows).sublime-keymap
@@ -99,12 +99,6 @@
			{ "key": "has_prev_field", "operator": "equal", "operand": true }
		]
	},
-	{ "keys": ["escape"], "command": "hide_panel", "args": {"cancel": true},
-		"context":
-		[
-			{ "key": "panel_visible", "operator": "equal", "operand": true }
-		]
-	},
	{ "keys": ["escape"], "command": "hide_overlay", "context":
		[
			{ "key": "overlay_visible", "operator": "equal", "operand": true }
@@ -353,13 +347,6 @@
	},

	// Auto-pair brackets
-	{ "keys": ["("], "command": "insert_snippet", "args": {"contents": "($0)"}, "context":
-		[
-			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
-			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
-			{ "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|;|\\}|$)", "match_all": true }
-		]
-	},
	{ "keys": ["("], "command": "insert_snippet", "args": {"contents": "(${0:$SELECTION})"}, "context":
		[
			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
@@ -383,13 +370,6 @@
	},

	// Auto-pair square brackets
-	{ "keys": ["["], "command": "insert_snippet", "args": {"contents": "[$0]"}, "context":
-		[
-			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
-			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
-			{ "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|;|\\}|$)", "match_all": true }
-		]
-	},
	{ "keys": ["["], "command": "insert_snippet", "args": {"contents": "[${0:$SELECTION}]"}, "context":
		[
			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
@@ -413,13 +393,6 @@
	},

	// Auto-pair curly brackets
-	{ "keys": ["{"], "command": "insert_snippet", "args": {"contents": "{$0}"}, "context":
-		[
-			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
-			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
-			{ "key": "following_text", "operator": "regex_contains", "operand": "^(?:\t| |\\)|]|\\}|$)", "match_all": true }
-		]
-	},
	{ "keys": ["{"], "command": "wrap_block", "args": {"begin": "{", "end": "}"}, "context":
		[
			{ "key": "indented_block", "match_all": true },
